### ISSUE TITLE
Colombien

### ISSUE THREAD

**Issue by marvelap** [iptv-org/iptv#651]
There is a possibility to add the Caracol and RCN channels of Colombia and the Winsport sports channel.  thank you very much

**Comment by freearhey** [comment]
Added Caracol and RCN channels to the playlist.

**Comment by marvelap** [comment]
Hi, Aleksandr

Thank you very much for the channels in Colombia, at the moment they are not working, but thank you very much. You have the new winsport+ from Colombia, I thank you, I am not a dealer, it is for personal use, I live in Canada

Luis
________________________________
De: Aleksandr Statciuk <notifications@github.com>
Enviado: miércoles, 29 de enero de 2020 6:59 p. m.
Para: iptv-org/iptv <iptv@noreply.github.com>
Cc: marvelap <luchoaugustop@hotmail.com>; Author <author@noreply.github.com>
Asunto: Re: [iptv-org/iptv] Colombien (#651)


Added Caracol and RCN channels to the playlist.

**Comment by marvelap** [comment]
Thank you very much for the channels in Colombia, at the moment they are not working, but thank you very much. You have the new winsport+ from Colombia, I thank you, I am not a dealer, it is for personal use, I live in Canada

**Comment by freearhey** [comment]
You're right, the channels aren't working anymore for some reason. We'll probably have to remove them, but as soon as there are working versions, I will return them. 

As for the Winsport channel, I haven't found it yet.

**Comment by marvelap** [comment]
thank you very much for your contribution
